src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/index.tsx

```diff
@@ -138,78 +145,97 @@ class Filter extends Component<Props> {
           return (
             <Responsive>
               {({ xs, sm, md }) => {
+                const hideTopBorder = this.props.hideTopBorder || xs
+
+                // Filters component to be rendered in Sidebar (desktop) and
+                // ActionSheet (mobile)
+                const Filters = () => {
```

Comment:
This is defined here so as to capture the scope without having to pass arguments around

Reply:
This file is becoming a bit unwieldy tho; in the next PR i'm going to break it up a bit (if it makes sense to).